source/client-side-encryption/client-side-encryption.rst

```diff
@@ -1332,6 +1367,67 @@ on sockets. See the SDAM spec description of `cooldownMS <../source/server-disco
 Because single threaded drivers may exceed ``serverSelectionTimeoutMS`` by the
 duration of the topology scan, ``connectTimeoutMS`` is also reduced.
 
+What's the deal with metadataClient, keyVaultClient, and the internal client?
+-----------------------------------------------------------------------------
+
+When automatically encrypting a command, the driver runs:
+- a ``listCollections`` command to determine if the target collection
+has a remote schema. This uses the ``metadataClient``.
+- a ``find`` against the key vault collection to fetch keys. This uses the
+``keyVaultClient``.
+
+**Why not reuse the parent MongoClient?**
```

Comment:
Funny character?

Reply:
It's a `?`. I'm not sure why it appears that way in the diff. Either way, I changed this to be a sub-heading instead of bold, so it can be linked to.